changelog/unreleased/enhancement-autocomplete-description

```diff
@@ -0,0 +1,5 @@
+Enhancement: Add description message to autocomplete component
+
+The description message will basically replace the informative text which was displayed via placeholder. It is still possible to use a placeholder though.
```

Comment:
"Still possible to use a placeholder" but prop got deleted?

Reply:
Yes because of `v-bind="additionalAttributes"`. It basically returns additional attributes + attributes which are being passed to the element. Hence the placeholder property does not need to be defined. I guess it's a classical question about "Do we keep it implicit oder explicit?" :D

Reply:
Maybe change the wording from `It is still possible to use a placeholder though.` to `While it's still possible to use a placeholder if you need it, we encourage you to not use it anymore.`. A label is always required, a description message can be added if more context is needed.